### PR TITLE
csputils: replace mp_colorspace with pl_color_space

### PR DESCRIPTION
**Pull request by kasper93**
Based on top of #12800

I did some of the dirty job...

### CODE REVIEW

File: video/out/vo_lavc.c [hunk 1]
```diff
@@ -23,6 +23,10 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#include <pthread.h>
```
Comment (kasper93) [inline video/out/vo_lavc.c:26]: Note to self, remove this? Where does it even came from?

Comment (kasper93) [reply video/out/vo_lavc.c]: removed

**Comment by kasper93** [comment]
Any news about this?

**Review by Dudemanguy — APPROVED** [review]
Late at night so beware but LGTM

**Comment by Dudemanguy** [comment]
Well let's see what ~breaks~ happens.

**Pull request merged**